dashboards/redmine/redmine.json

```diff
@@ -0,0 +1,1293 @@
+			}
+		  },
+		  {
+			"title": "Applications (click on the application name to filter)",
+			"layout": {
+			  "column": 3,
+			  "row": 1,
+			  "width": 4,
+			  "height": 4
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.bar"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "SELECT count(*) from Metric facet appName"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  }
+			}
+		  },
+		  {
+			"title": "Spent Time On Issues (hours)",
+			"layout": {
+			  "column": 7,
+			  "row": 1,
+			  "width": 4,
+			  "height": 4
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.area"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "legend": {
+				"enabled": true
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "SELECT count(newrelic.timeslice.value) AS `hour` FROM Metric WHERE metricTimesliceName like '%Custom/issuesSpentHours%'   TIMESERIES "
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  }
+			}
+		  },
+		  {
+			"title": "Average Page Load (seconds)",
+			"layout": {
+			  "column": 11,
+			  "row": 1,
+			  "width": 2,
+			  "height": 2
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.billboard"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT AVERAGE(newrelic.goldenmetrics.browser.application.pageLoadSeconds) AS ''"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  }
+			}
+		  },
+		  {
+			"title": "",
+			"layout": {
+			  "column": 1,
+			  "row": 3,
+			  "width": 2,
+			  "height": 2
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.markdown"
+			},
+			"rawConfiguration": {
+			  "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/launcher/catalog-pack-details.launcher/?pane=eyJuZXJkbGV0SWQiOiJjYXRhbG9nLXBhY2stZGV0YWlscy5jYXRhbG9nLXBhY2stY29udGVudHMiLCJxdWlja3N0YXJ0SWQiOiIxMzU3ZDA0Mi0xZjViLTQ4NzEtYTE0YS1lMmRiMTBiZWM3OTUifQ==).\n\nUnable to find data in your dashboard? - [Troubleshoot here](\nhttps://docs.newrelic.com/docs/apm/agents/ruby-agent/troubleshooting/no-data-appears-ruby/)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=Redmine) here and let us know how we can improve it for you.\n"
```

Comment:
Please add documentation link on "Add Data" instead of ruby installation

Reply:
@pkosanam Updated the changes